src/pages/workspaces/workspace/Data.js

```diff
@@ -501,6 +502,11 @@ const WorkspaceData = _.flow(
                     selected: _.isEqual(selectedDataType, [snapshotName, tableName]),
                     onClick: () => {
                       setSelectedDataType([snapshotName, tableName])
+                      Ajax().Metrics.captureEvent(Events.workspaceSnapshotContentsView, {
+                        ...extractWorkspaceDetails({ workspace }),
+                        snapshotName,
+                        entityType: tableName
+                      })
```

Comment:
could you include the snapshotId here? That is a more stable identifier than the snapshot name, since name is user-editable